modules/serverless-logic-tracing-http-requests-openAPI.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-tracing-http-requests-openAPI_{context}"]
+= Tracing HTTP requests and responses
+
+To trace HTTP requests and responses, OpenShift Serverless Logic uses the Apache HTTP Client. You can enable tracing by setting the following property:
+
+[source,terminal]
```

Comment:
This is also the first step in the procedure on line 26. I recommend removing this code block entirely.